apps/aeutil/lib/serialization.ex

```diff
@@ -266,6 +266,12 @@ defmodule Aeutil.Serialization do
   end
 
   defp serialize_txs_info_to_json([h | t], acc) do
+    tx_hash =
```

Comment:
either rename this, as it is not the hash or include hashing with a pipe

Reply:
Also you should call some `hash_tx` function here so hashing is defined in one place. I think we have `SignedTx.hash_tx`